.github/workflows/dependabot-auto-merge.yaml

```diff
@@ -17,7 +17,7 @@ jobs:
           github-token: '${{ secrets.GITHUB_TOKEN }}'
       - name: Enable auto-merge for Dependabot PRs
         if: ${{ steps.metadata.outputs.update-type == 'version-update:semver-patch' }}
-        run: gh pr merge --auto "$PR_URL"
+        run: gh pr merge --auto --squash "$PR_URL"
```

Comment:
`--merge` or `--rebase` or `--squash`.
Added because it was a mandatory option.